maintainers/maintainer-list.nix

```diff
@@ -6653,6 +6653,12 @@
     githubId = 7677321;
     name = "Paul Trehiou";
   };
+  nyanotech = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add nyanotech``.